theano/tensor/tests/test_basic.py

```diff
@@ -1824,14 +1826,43 @@ def test_py_c_match():
     inplace=True,
     skip=skip_scipy)
 
+_good_broadcast_binary_polygamma = dict(
+    normal=(randint_ranged(0, 3, (2, 3)), rand_ranged(1, 10, (2, 3)),),
```

Comment:
I think it can be a tensor.  This works:
>>> scipy.special.polygamma([[1,2],[3,2]], [[1.5,2.5],[3.5,2.5]])
array([[ 0.9348022 , -0.23620405],
       [ 0.07030585, -0.23620405]])